### PR TITLE
add a comment about bumping CRD versions

### PR DESCRIPTION
**Pull request by roytman**
Signed-off-by: Alexey Roytman <roytman@il.ibm.com>

### CODE REVIEW

File: docs/BackwardCompatibility.md [hunk 1]
```diff
@@ -121,13 +121,18 @@ possible CRD changes without changing its schema. And in the future will check p
 #### CRD changes with None conversation strategy
 When we use the None conversation strategy, we are able to change a required field in version v1 to an optional field in 
 v2. In addition, we are able to add a new  optional field in v2. These changes are inline with compatibility examples 
-from [\[5\]](#references)
+from [\[5\]](#references).
+
+When we do these None conversation strategy changes we don't have to bump a CRD version, see discussion in the Kubernetes
```
Comment (shlomitk1) [inline docs/BackwardCompatibility.md:126]: conversion strategy, not conversation

File: docs/BackwardCompatibility.md [hunk 2]
```diff
@@ -237,5 +242,6 @@ requirements<a name="1"></a>
 3. Kubernetes "[Serving multiple versions of a CRD](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#serving-multiple-versions-of-a-crd)"<a name="3"></a>
```
Comment (shlomitk1) [inline docs/BackwardCompatibility.md:242]: The mentioned link presents another link: https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/

File: docs/BackwardCompatibility.md (hunk 1 above)
Comment (roytman) [reply docs/BackwardCompatibility.md:126]: fixed, thanks

File: docs/BackwardCompatibility.md (hunk 2 above)
Comment (roytman) [reply docs/BackwardCompatibility.md:242]: removed

File: docs/BackwardCompatibility.md [hunk 3]
```diff
@@ -121,13 +121,18 @@ possible CRD changes without changing its schema. And in the future will check p
 #### CRD changes with None conversation strategy
 When we use the None conversation strategy, we are able to change a required field in version v1 to an optional field in
```
Comment (shlomitk1) [inline docs/BackwardCompatibility.md:122]: conversion (change all occurrences)

Comment (roytman) [reply docs/BackwardCompatibility.md]: all fixed, thanks

**Review by shlomitk1 — APPROVED** [review]

**Pull request merged**